docs/user/metal/install_ipi.md

```diff
@@ -50,7 +50,7 @@ accessible over the external network which may not be desirable.
 
 * **NIC #2 - Provisioning Network (optional) **
   * A private network used for PXE based provisioning.
-  * You must specify `provisioningNetworkInterface` to indicate which
+  * You can specify `provisioningNetworkInterface` to indicate which
```

Comment:
Should we mention that if not specified the interface is derived from the `bootMACAddress`?

Reply:
Should we also mention that when provisioningNetwrokInterface is specified, the user needs to make sure all the control plane hosts have an interface of that name?

Reply:
thanks, updated. Apologies for the slow response.